maven-wrapper/src/test/java/org/apache/maven/wrapper/WrapperExecutorTest.java

```diff
@@ -222,27 +226,27 @@ public void testEnvironmentVariableOverwrite_mvnwRepoUrl_trailingSlash()
 
     WrapperExecutor wrapper = prepareWrapperExecutorWithEnvironmentVariables(environmentVariables);
 
-    Assert.assertEquals( "https://repo/test/path/to/bin.zip", wrapper.getDistribution().toString() );
+    assertEquals( "https://repo/test/path/to/bin.zip", wrapper.getDistribution().toString() );
   }
 
   @Test
-  public void testEnvironmentVariableOverwrite_packageName()
-          throws Exception
+  void testEnvironmentVariableOverwrite_packageName()
+    throws Exception
   {
     final Map<String, String> environmentVariables = new HashMap<>();
     environmentVariables.put( MVNW_REPOURL, "https://repo/test" );
     properties = new Properties();
-    properties.put( "distributionUrl", "https://server/org/apache/maven/to/bin.zip" );
-    writePropertiesFile( properties, propertiesFile, "header" );
+    properties.put("distributionUrl", "https://server/org/apache/maven/to/bin.zip");
+    writePropertiesFile(properties, propertiesFile, "header");
```

Comment:
Style: you removed whitespace after the opening parenthesis

Reply:
Sorry about that; all these changes are automated, not sure why only here the whitespace handling was off.